docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -511,17 +543,16 @@ public class BearerTokenAuthorizationTest {
 }
 ----
 
-Note that the `quarkus-test-oidc-server` extension includes a signing RSA private key file in a `JSON Web Key` (`JWK`) format and points to it with a `smallrye.jwt.sign.key.location` configuration property. It allows to use a no argument `sign()` operation to sign the token.
-
-Testing your `quarkus-oidc` `service` application with `OidcWiremockTestResource` provides the best coverage as even the communication channel is tested against the Wiremock HTTP stubs.
-`OidcWiremockTestResource` will be enhanced going forward to support more complex bearer token test scenarios.
+The `quarkus-test-oidc-server` extension includes a signing RSA private key file in a `JSON Web Key` (`JWK`) format and points to it with a `smallrye.jwt.sign.key.location` configuration property.
+It allows you to sign the token by using a no-argument `sign()` operation.
 
-If there is an immediate need for a test to define Wiremock stubs not currently supported by `OidcWiremockTestResource`
-one can do so via a `WireMockServer` instance injected into the test class, for example:
+Testing your `quarkus-oidc` `service` application with `OidcWiremockTestResource` provides the best coverage because even the communication channel is tested against the WireMock HTTP stubs.
+It is anticipated that `OidcWiremockTestResource` will be enhanced in an upcoming release to support more complex bearer token test scenarios.
```

Comment:
It makes a specific commitment. I propose to drop this sentence completely

Reply:
@sberyozkin: Per your suggestion, I will remove this statement.
Just FYI, according to Red Hat guidelines, it is possible to formulate details about a possible future enhancement in this way by using phrasing such as "anticipate" or "expect" and not stating a specific future release.

Reply:
@sheilamjones Sure, will keep it in mind, as far as `OidcWiremockTestResource` is concerned, I thought, at the time of its creation by the contributor that we'd keep evolving it, we may still do add some bits to it, but since another contribution came to support local wiremocks and enhancing `OidcWiremockTestResource` has become more or less unnecessary with some rare exceptions